lostfiles.py

```diff
@@ -116,11 +312,37 @@ def parse_args() -> argparse.Namespace:
     )
     return parser.parse_args()
 
+def installed_packages():
+    for pkg, dirs in PKG_PATHS.items():
+        if package_exist(pkg):
+            for dir in dirs:
```

Comment:
`dir` shadows the built-in function `dir()` which can cause problems, please rename it to something else

Reply:
Change a directory